test/integration/spec/docker/reconnection.js

```diff
@@ -143,30 +162,72 @@ describe('Reconnection states and events', function() {
     assert.equal(typeof isRunningInsideDocker, 'boolean');
   });
 
-  it('can detect media flow', async () => {
-    const rooms =  await setup(2);
-    await waitFor(rooms.map(validateMediaFlow), 'validate media flow', VALIDATE_MEDIA_FLOW_TIMEOUT);
-    rooms.forEach(room => room.disconnect());
-    return completeRoom(rooms[0].sid);
+  context('should be able to', () => {
+    let rooms;
+
+    before(async function() {
+      if (!isRunningInsideDocker) {
+        // eslint-disable-next-line no-invalid-this
+        this.skip();
+      } else {
+        rooms = await setup(['Alice', 'Bob', 'Charlie'], {
+          iceTransportPolicy: 'relay'
+        }, DOCKER_PROXY_TURN_REGIONS);
+      }
+    });
+
+    after(async () => {
+      if (rooms) {
+        rooms.forEach(room => room.disconnect());
+        await completeRoom(rooms[0].sid);
+        rooms = null;
+      }
+    });
+
+    it('validate media flow', () => {
+      return waitFor(rooms.map(validateMediaFlow), 'validate media flow', VALIDATE_MEDIA_FLOW_TIMEOUT);
```

Comment:
I have found that including room sid in messages helps debug. Especially comes handy when debugging failed tests on circleCI. Suggest: 
`return waitFor(rooms.map(validateMediaFlow), 'validate media flow: ${rooms[0].sid}', VALIDATE_MEDIA_FLOW_TIMEOUT);`

Reply:
This comment applies for pretty much all `waitFor` usage.